sycl/test-e2e/Matrix/joint_matrix_bfloat16_colmajorA_colmajorB.cpp

```diff
@@ -13,8 +15,6 @@
 // This tests support of col major layout for matrix B which does transpose and
 // then VNNI transform. This is currently only available on AMX
 
-// XFAIL: gpu
```

Comment:
Does this test pass on PVC? I did not see any unexpected pass on PVC for this test...